rocketchat-common/src/main/java/com/rocketchat/common/data/lightstream/document/RocketChatRolesDocument.java

```diff
@@ -0,0 +1,88 @@
+package com.rocketchat.common.data.lightstream.document;
+
+import java.util.Date;
+import org.json.JSONObject;
+
+public class RocketChatRolesDocument {
+    String name;
+    String scope;
+    String description;
+    Boolean _protected;
+    Date updatedAt;
+
+    public RocketChatRolesDocument(JSONObject roles) {
+        name = roles.optString("name");
+        scope = roles.optString("scope");
+        description = roles.optString("description");
+        _protected = roles.optBoolean("protected");
```

Comment:
Why use a underscore on the variable name?

Reply:
I think it is a one of the keyword, I cannot have variable name same as java keyword.

Reply:
Maybe change to `isProtected`?